R/args.R

```diff
@@ -34,9 +34,16 @@ CmdStanArgs <- R6::R6Class(
                           seed = NULL,
                           init = NULL,
                           refresh = NULL,
-                          output_dir = NULL,
-                          validate_csv = TRUE) {
-
+                          dir = NULL,
+                          validate_csv = TRUE,
+                          # deprecated
+                          output_dir = NULL) {
+      # temporary deprecation warnings
+      if (!is.null(output_dir)) {
+        warning("'output_dir' is deprecated. Please use 'dir' instead.")
+        dir <- output_dir
+      }
```

Comment:
Do we need this in args.R if we also have it in each of the mode fitting methods? Is there a  case where we only throw this warning here without also throwing it from, e.g., `$sample()`?

Reply:
Oh right. Let me double check.

Reply:
Actually yeah, I just removed it. cmdstanArgs is internal. No need to keep it.